.jenkins

```diff
@@ -145,4 +190,16 @@ pipeline {
             }
         }
     }
+    post {
+        always {
+            node('docker') {
+                recordIssues(
+                    enabledForFailure: true,
+                    tools: [cmake(), gcc(), clang(), clangTidy()],
+                    qualityGates: [[threshold: 1, type: 'TOTAL', unstable: true]],
+                    filters: [excludeFile('/usr/local/cuda.*')]
```

Comment:
This is not ideal but I am not sure how to to make it a system header path with CMake.  The path is implicitly added by NVCC.